content/blog/2023-05-16-comparison-of-ethereum-classic-and-ethereum-monetary-policies/index.md

```diff
@@ -0,0 +1,146 @@
+
+The way it works is that miners in the blockchain create blocks of transactions and to stamp them cryptographically they need to do a lot of computational work, which uses a lot of electricity. Then, when the blocks are accepted by the rest of the network, the miners are credited newly issued coins by the algorithm.
+
+The above system guarantees that ETC will be sound money as gold was in the old days, hence the term “digital gold”.
+
+ETH is not digital gold because it is a proof of stake blockchain and validators do not have to do any work to produce blocks or earn rewards. They just receive transactions, package them into batches they call blocks to steal some lure from proof of work blockchains, and they vote on them to affirm their validity.
+
+The above means that to produce the coin in Ethereum is practically costless, thus the currency cannot be compared to gold.
+
+The best comparison that may be made for ETH is that it is very similar to fiat money. Fiat money is produced by central banks with no cost whatsoever, thus ETH may be called community fiat money as it is printed by a private community rather than a government. 
+
+## Algorithmic
+
+ETC’s monetary policy is algorithmic because in 2017 it migrated to a schedule that is set in stone as it establishes the whole future distribution in the software in the form of a supply algorithm.
+
+The supply algorithm is a set of simple and transparent mathematical rules that establish what number of coins will be paid to miners per block, the frequency of the payments, and a schedule that will decrease these payments as time passes by.
+
+ETH’s monetary policy is manual, not algorithmic, because it has been changed several times and will likely change many times more in the future given that proof of stake tends to centralization.
+
+The fact that it has a software algorithm that determines payments to validators does not mean it is algorithmic as long as developers may change the rules whenever they please.
+
+## Capped Supply
+
+ETC has a capped supply of its coin. This means that it’s known what is the total amount of ETC that will ever exist.
+
+Given Ethereum Classic’s frequency of payments to miners, the number of coins it issues per block, and its discount schedule, it will have a maximum supply of ETC of 210,700,000.
+
+ETH does not have a capped supply, but has a two way system with an issuance schedule and a coin burning mechanism. These two combined sometimes produce a net issuance of coins and sometimes a net redemption of coins. It all depends on the amounts of staked ETH by validators and the levels of transaction fees.
+
+However, whether Ethereum has a capped supply or not , or if they burn coins or not, is inconsequential in a centralized network with a manually managed monetary policy because they may change the rules at any time. 
```

Comment:
Line 69: Spacing typo
```suggestion
However, whether Ethereum has a capped supply or not, or if they burn coins or not, is inconsequential in a centralized network with a manually managed monetary policy because they may change the rules at any time. 
```